classes/controller/AdminController.php

```diff
@@ -2705,8 +2705,9 @@ public function setMedia($isNewTheme = false)
             // add Jquery 3 and its migration script
             $this->addJs(_PS_JS_DIR_ . 'jquery/jquery-3.4.1.min.js');
             $this->addJs(_PS_JS_DIR_ . 'jquery/jquery-migrate-3.1.0.min.js');
-            // implement $.browser object, that has been removed since jquery 1.9
+            // implement $.browser object and live method, that has been removed since jquery 1.9
             $this->addJs(_PS_JS_DIR_ . 'jquery/jquery.browser-0.1.0.min.js');
+            $this->addJs(_PS_JS_DIR_ . 'jquery/jquery.live.js');
```

Comment:
ah yes, good point !
This PR is in wip so far because it's not finished, and the jquery.live file will be replaced by the one I pushed on a dedicated repo.

But thanx for the comment idea, totally forgot about that.

Reply:
Sorry for the WIP :stuck_out_tongue:

Reply:
In the end I implementend your suggestion @Progi1984 :D . This js file is not to be fetched via npm in the legacy backoffice.